packages/orun/orun.0.1.2/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Run benchmarks and measure performance"
+maintainer: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+authors: "Stephen Dolan <stephen.dolan@cl.cam.ac.uk>"
+license: "MIT"
+dev-repo: "git+https://github.com/ocaml-bench/orun.git"
+depends: [ "ocaml" "cmdliner" "yojson" ]
+build: [
+  ["ocaml-update-c" "wait4.c"] {ocaml:update-c}
```

Comment:
I'm not sure of what `{<package>:update-c}` means. I don't see this anywhere in the documentation.